.github/workflows/ci.yml

```diff
@@ -0,0 +1,66 @@
+name: Continuous Integration
+
+on: [ push ]
+
+jobs:
+  lint:
+    strategy:
+      matrix: 
+        os: [ 'macos-latest' ]
+        go: [ '1.13' ]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-go@v1
+        with:
+          go-version: ${{ matrix.go }}
+ 
+      - run: gofmt 2>&1
```

Comment:
I think you've removed too much? If you just have `gofmt 2>&1` won't it just hang there waiting until it finds something on stdin? I'd expect this to look something like:

```
gofmt -l $(find . -type f -name '*.go') 2>&1
```

Also, I guess `goimports` is not available? If not, we can use `gofmt` for now.

Reply:
I just took a look at the run in your repo, and it looks like it didn't hang waiting for input, but it did finish in 0s (and didn't seem to do anything). This might be because GitHub connects an empty input stream to the job?

https://github.com/jamesmoriarty/alpaca/runs/398642375

One way to test this would be to push a commit with some bad formatting, and see if it complains.

Reply:
Thanks for catching this. Updated e.g. https://github.com/jamesmoriarty/alpaca/runs/400039825